modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/internal/events/ClientExtensionsServicePreAction.java

```diff
@@ -117,6 +112,36 @@ private CET _getCET(
 			companyId, clientExtensionEntryRel.getCETExternalReferenceCode());
 	}
 
+	private ThemeCSSCET _getControlPanelThemeCSSCET(Layout layout) {
+		if (!FeatureFlagManagerUtil.isEnabled("LPD-34650")) {
+			return null;
+		}
+
+		List<ClientExtensionEntryRel> clientExtensionEntryRels =
+			_clientExtensionEntryRelLocalService.getClientExtensionEntryRels(
+				_portal.getClassNameId(Layout.class), layout.getPlid(),
+				ClientExtensionEntryConstants.TYPE_THEME_CSS);
+
+		if ((clientExtensionEntryRels == null) ||
+			(clientExtensionEntryRels.size() != 1)) {
+
+			return null;
+		}
+
+		ClientExtensionEntryRel clientExtensionEntryRel =
+			clientExtensionEntryRels.get(0);
+
+		CET cet = _cetManager.getCET(
+			layout.getCompanyId(),
+			clientExtensionEntryRel.getCETExternalReferenceCode());
+
+		if (cet == null) {
+			return null;
+		}
+
+		return (ThemeCSSCET)cet;
```

Comment:
Is this null check needed? If `cet` is null, `(ThemeCSSCET) cet` would return null anyway, right?